### ISSUE TITLE
uglify-es is deprecated

### ISSUE THREAD

**Issue by saschanaz** [w3c/respec#1902]
https://github.com/mishoo/UglifyJS2/issues/3156#issuecomment-392943058

>`uglify-es` is no longer maintained.

😮

**Comment by marcoscaceres** [comment]
Whaaaaa!?! Why did that happen?

**Comment by marcoscaceres** [comment]
Ok, looking at https://github.com/terser-js/terser

**Comment by saschanaz** [comment]
terser is just a fork so I'm expecting the migration be easy enough.

**Comment by marcoscaceres** [comment]
yeah, that's what I'm reading too... I wonder if it matters that we depend on a particular version of Uglify-es still... the newer version had a bug that prevented us from upgrading.

**Comment by marcoscaceres** [comment]
Ok, looks like it should work, as they've kept the same release numbers and everything... and they might actually have fixed the bug (I don't recall anymore what the bug was), given that they've merged some things (🤞).

**Comment by saschanaz** [comment]
I tried uglify-es 3.3.9 again (which is the last version that caused #1461) and the issue seems to be magically fixed already. That bug magically appeared anyway.

**Comment by marcoscaceres** [comment]
Alright! Let's switch it up!!!! :D TERSER TERSER TERSER!!!
